### PR TITLE
Make the project devup and gb compatible

### PR DESCRIPTION
**Pull request by csgregorian**
@Shopify/traffic

Migrated from Godeps to gb, made compatible with devup.

### CODE REVIEW

**Comment by csgregorian** [comment]
Dependency paths corrected, methods are consistent with current dependencies.

@Shopify/traffic

File: circle.yml [hunk 1]
```diff
@@ -1,17 +1,19 @@
 dependencies:
   pre:
-    - go get github.com/tools/godep
-    - bundle install --without development
+    - go get github.com/constabulary/gb/...
+    - gem install package_cloud
```
Comment (ElvinEfendi) [inline circle.yml]: You can move this to the Gemfile to have all the gems in one file.

**Comment by klautcomputing** [comment]
image

`dev up` and `bin/007` work right after cloning it 👍

**Review by klautcomputing — APPROVED** [review]
One tiny thing, please set your editor to include newlines and the end of the file.

**Review by ElvinEfendi — CHANGES_REQUESTED** [review]
👍 Looks good. Couple of minor things only:
I think this `bin/007` should not be in the repo, as you ignore the `bin` folder.

Also before merging to master please deploy this branch to a host and run it for a while to make sure using the newer version `goprocinfo` did not change the program behaviour/output.

**Comment by csgregorian** [comment]
image

Confirmed package_cloud still installs.

**Comment by csgregorian** [comment]
image
:tophat:

@ElvinEfendi

**Review by ElvinEfendi — APPROVED** [review]
Squash and merge 👍

**Pull request merged**